packages/server/tests/utils.ts

```diff
@@ -50,10 +50,19 @@ const supportedEnvs = ['previewnet', 'testnet', 'mainnet'];
 export default class TestUtils {
```

Comment:
I think that we should introduce `AssertionUtils` class which contains utils only for assertions and have another class for `InteractionUtils` class that contains functions like:
`executeQuery`, `executeTransaction`, `executeAndGetTransactionReceipt` .. etc

Reply:
Let's do that in a separate PR, currently there are a lot of open PRs and doing a refactor like this will make merging them very complicated